app/components/ui/Dashboard.tsx

```diff
@@ -1,12 +1,14 @@
 import React, { useRef, useState } from "react";
 import GraphCard from "./GraphCard";
 import { Household } from "@/app/models/Household";
+import { formType } from "@/app/schemas/formSchema";
 
 interface DashboardProps {
-  data: Household;
+  processedData: Household;
+  inputData: formType; // Add this property
 }
 
-const Dashboard: React.FC<DashboardProps> = ({ data }) => {
+const Dashboard: React.FC<DashboardProps> = ({ inputData }) => {
```

Comment:
```suggestion
const Dashboard: React.FC<DashboardProps> = () => {
```

This would avoid needing the work around for an unused variable.